base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
No one is going to write code testing against "NT." Formal kernel names are internal implementation details that don't matter so much at the Julia level - the userland OS, environment, and common names that people will want to test against and know what it means even if they've never written a shell script that called `uname`, do matter.